CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Added flake8 checks to the lint.yml [#321](https://github.com/scanapi/scanapi/pull/321)
```

Comment:
I believe we don't need a changelog entry here, since it is only an internal change:  https://github.com/scanapi/scanapi/wiki/Changelog#what-warrants-a-changelog-entry

Reply:
alright, I will remove this.